deeplake/htype.py

```diff
@@ -66,7 +69,7 @@ class htype:
     htype.BBOX: {"dtype": "float32", "coords": {}, "_info": ["coords"]},
     htype.BBOX_3D: {"dtype": "float32", "coords": {}, "_info": ["coords"]},
     htype.AUDIO: {"dtype": "float64"},
-    htype.EMBEDDING: {"dtype": "float32"},
+    htype.EMBEDDING: {},
```

Comment:
Why did we decide to make it empty?

Reply:
It was already empty, see below. There was a duplicate.